Assets/MixedRealityToolkit/Utilities/BuildAndDeploy/BuildDeployPreferences.cs

```diff
@@ -26,8 +52,13 @@ public static class BuildDeployPreferences
         /// </remarks>
         public static string BuildDirectory
         {
-            get => $"{EditorPreferences.Get(EDITOR_PREF_BUILD_DIR, "Builds")}/{EditorUserBuildSettings.activeBuildTarget}";
-            set => EditorPreferences.Set(EDITOR_PREF_BUILD_DIR, value.Replace($"/{EditorUserBuildSettings.activeBuildTarget}", string.Empty));
+            get => $"{_savedPreferences.BuildDirectory}/{EditorUserBuildSettings.activeBuildTarget}";
+            set
+            {
```

Comment:
You should look at PR #6315 for related task/overlap

Reply:
oh sweet!  Are there some guidance on how to use that feature?  How long would it take for that PR to get pushed?

Reply:
Not exactly, I literally just created it. I actually didn't include documentation in that PR because
1) It's atm primarily for MRTK preferences
2) I'd rather the feature go through some trials and tribulations before being publicly mentioned

I'm trying to do more testing and once it's reviewed/approved it can go in. So I guess a few days?

You can pull the branch yourself and try it out. See if you find any bugs or issues

Reply:
@genbod, FYI #6315 was merged into MRKT_Dev